README.md

```diff
@@ -17,9 +16,12 @@ If you have any questions, feel free to join my Discord
 [![Video](https://i.imgur.com/1R5uOnA.png)](https://youtu.be/ReE9nVuMCSE)
 
 ## Download
+
 <a href="https://flathub.org/apps/details/com.github.polymeilex.neothesia"><img width="240" alt="Download on Flathub" src="https://flathub.org/assets/badges/flathub-badge-en.png"/></a>
 
-Or binary releases:
+Arch Linux(AUR): <https://aur.archlinux.org/packages/neothesia>
```

Comment:
Don't take this personally, but could we mention that the code of the package is not audited by Neothesia, for example `(Unofficial, maintained by @zayn7lie)` or something along those lines.

Feel free to word it however you like. I just want to indicate to people that perhaps they should read the script before installing.

Reply:
Sure (Sorry, I forget about this~)